mk/cfg/mips-unknown-linux-musl.mk

```diff
@@ -0,0 +1,24 @@
+# mips-unknown-linux-musl configuration
+CC_mips-unknown-linux-musl=mips-openwrt-linux-gcc
```

Comment:
TODO change the prefixes to something vendor neutral. Probably `mips-linux-musl-gcc`.